quodlibet/quodlibet/_main.py

```diff
@@ -118,7 +118,16 @@ def get_user_dir():
     if os.name == "nt":
         USERDIR = os.path.join(windows.get_appdate_dir(), "Quod Libet")
     else:
```

Comment:
We don't want this change on macOS (quodlibet.util.is_osx), needs a new case using the old path for this.